contracts/dapis/DapiServer.sol

```diff
@@ -572,23 +575,15 @@ contract DapiServer is
     /// a standard implementation of Airnode is being used, these can be
     /// expected to be correct. Either way, the assumption is that it does not
     /// matter for the purposes of a Beacon set update subscription.
-    /// @param subscriptionId Subscription ID
-    /// @param airnode Airnode address
-    /// @param relayer Relayer address
-    /// @param sponsor Sponsor address
-    /// @param timestamp Timestamp used in the signature
     /// @param data Fulfillment data (an `int256` encoded in contract ABI)
-    /// @param signature Subscription ID, timestamp, sponsor wallet address
-    /// (and fulfillment data if the relayer is not the Airnode) signed by the
-    /// Airnode wallet
     function fulfillPspBeaconSetUpdate(
-        bytes32 subscriptionId, // solhint-disable-line no-unused-vars
-        address airnode, // solhint-disable-line no-unused-vars
-        address relayer, // solhint-disable-line no-unused-vars
-        address sponsor, // solhint-disable-line no-unused-vars
-        uint256 timestamp, // solhint-disable-line no-unused-vars
+        bytes32,
+        address,
+        address,
+        address,
+        uint256,
         bytes calldata data,
-        bytes calldata signature // solhint-disable-line no-unused-vars
```

Comment:
And we can even keep the argument names like this for reference:

```
    function fulfillPspBeaconSetUpdate(
        bytes32 /*subscriptionId*/,
        address /*airnode*/,
        address /*relayer*/,
        address /*sponsor*/,
        uint256 /*timestamp*/,
        bytes calldata data,
        bytes calldata /*signature*/
```

Reply:
and this